### ISSUE TITLE
Error with sync

### ISSUE THREAD

**Issue by xstefen** [OnePlusOSS/android#43]
Though I am more interested in my devices source, fajita/6T, when attempting to sync the oneplus6/9.0.0 branch I am given:

git@privgit.codeaurora.org: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by TeamFahQ** [comment]
I can confirm. It seems that branch oneplus6/9.0.0 branch has yet to be fully open sourced. 

I have found that the sources in oneplus6/9.0.0 branch for the manifest are actually released in open source trees. But the problem is, the paths, urls, repos, etc are not correct. I have fixed them here
[https://github.com/oneplus6t-lz/android/commit/d78b6f53799bf7d813f417cae4eb3aba6e5926ab](url)
But there is another issue that causes repo sync to fail.
It appears branch oneplus6/9.0.0 in OnePlusOSS/android_frameworks_base does not exist.

Please fix per GPL/GNU.

**Comment by xstefen** [comment]
Dear @OnePlusOSSAdmin ,
Please do not ignore. Patiently awaiting response still.

**Comment by xstefen** [comment]
Emails, github issues, and forum posts all continue to be ignored. What are you hiding OnePlus?

**Comment by xstefen** [comment]
> Dear @OnePlusOSSAdmin ,
> Please do not ignore. Patiently awaiting response still.

**Comment by TeamFahQ** [comment]
I don't think they are listening, or they just don't care. Not only are we missing the 9.0 branch for frameworks/base, but they still haven't uploaded device trees.

**Comment by xboxfanj** [comment]
They accidentally pushed their private manifest.   They fixed most proprietary projects at https://github.com/OnePlusOSS/android/commit/79918d0d0009ed83db0655ce057a24eeb083ffd2

To fix the remaining issues, change https://github.com/OnePlusOSS/android/blob/oneplus6/9.0.0/default.xml#L3 to `<remote fetch="https://source.codeaurora.org/quic/la/" name="caf" review="codeaurora.org"/>` which will match public CAF.

**Comment by xstefen** [comment]
I believe its possible to get privgit access, however when I submitted for it I was denied due to an `@gmail.com` address. Haven't tried again yet. Thanks both for your replies here, at least someone is paying attention ;)

**Comment by xboxfanj** [comment]
It's only possible to get privgit access if you have a Qualcomm license. In this case though, the code is public anyway.  Privgit is only necessary when tags aren't public yet, such as 855 now or 845 before July last year. Otherwise, public and private are the same.

**Comment by xstefen** [comment]
> It's only possible to get privgit access if you have a Qualcomm license. In this case though, the code is public anyway. Privgit is only necessary when tags aren't public yet, such as 855 now or 845 before July last year. Otherwise, public and private are the same.

Good to know, thanks sir again

**Comment by TeamFahQ** [comment]
> They accidentally pushed their private manifest. They fixed most proprietary projects at [79918d0](https://github.com/OnePlusOSS/android/commit/79918d0d0009ed83db0655ce057a24eeb083ffd2)
> 
> To fix the remaining issues, change https://github.com/OnePlusOSS/android/blob/oneplus6/9.0.0/default.xml#L3 to `<remote fetch="https://source.codeaurora.org/quic/la/" name="caf" review="codeaurora.org"/>` which will match public CAF.

But that still doesn't fix the issue of not having branch 9.0.0 for frameworks/base, or device trees for the oneplus6(T) for that matter.
Do we use the base CAF frameworks/base and build it ourselves from there? But without the API's for OxygenOs, it would just be a base rom.
Should we start another issue thread on the frameworks/base repo to address the missing branch?
How can we start an issue thread for the missing device trees?

**Comment by xboxfanj** [comment]
I did not realize that.

@leonfish77101 can you take a look?

**Comment by xstefen** [comment]
> > They accidentally pushed their private manifest. They fixed most proprietary projects at [79918d0](https://github.com/OnePlusOSS/android/commit/79918d0d0009ed83db0655ce057a24eeb083ffd2)
> > To fix the remaining issues, change https://github.com/OnePlusOSS/android/blob/oneplus6/9.0.0/default.xml#L3 to `<remote fetch="https://source.codeaurora.org/quic/la/" name="caf" review="codeaurora.org"/>` which will match public CAF.
> 
> But that still doesn't fix the issue of not having branch 9.0.0 for frameworks/base, or device trees for the oneplus6(T) for that matter.
> Do we use the base CAF frameworks/base and build it ourselves from there? But without the API's for OxygenOs, it would just be a base rom.
> Should we start another issue thread on the frameworks/base repo to address the missing branch?
> How can we start an issue thread for the missing device trees?

Or over a month with zero reply, to email either, which I quote `If there is any advise or question of the released source code, please contact with oss.request@oneplus.com.` so you would at least expect something there.

**Comment by xstefen** [comment]
#43

**Comment by xstefen** [comment]
#43

**Comment by 311303** [comment]
I can't sync it, can I solve it?

**Comment by xstefen** [comment]
> I can't sync it, can I solve it
A bot gives more reply than 1+

**Comment by 311303** [comment]
> > I can't sync it, can I solve it
> > A bot gives more reply than 1+

Sorry, what's that?

**Comment by xstefen** [comment]
#43

**Comment by xstefen** [comment]
#43 Maybe one of these days we will get an answer. Still waiting on that reply email too....
"If there is any advise or question of the released source code, please contact with oss.request@oneplus.com."
Your company is ridiculous, I learned that with the bacon, I gave you another chance with the 6T despite the terrible history of mistakes and misleading from you, and it continues...

**Comment by aaronqiuchiu** [comment]
20f0a05dad43f397f64b9c0977bc1caa21eb5224
